PhysicsTools/NanoAOD/python/nanoDQM_cfi.py

```diff
@@ -108,11 +107,11 @@
                 Plot1D('area', 'area', 20, 2, 4, 'jet catchment area, for JECs'),
                 Plot1D('btagCMVA', 'btagCMVA', 20, -1, 1, 'CMVA V2 btag discriminator'),
                 Plot1D('btagCSVV2', 'btagCSVV2', 20, -1, 1, ' pfCombinedInclusiveSecondaryVertexV2 b-tag discriminator (aka CSVV2)'),
-                Plot1D('btag_DDBvL', 'btag_DDBvL', 20, 0, 1, 'DeepDoubleX (mass-decorrelated) discriminator for H(Z)->bb vs QCD'),
+                Plot1D('btagDDBvL', 'btagDDBvL', 20, inf, -inf, 'DeepDoubleX (mass-decorrelated) discriminator for H(Z)->bb vs QCD'),
```

Comment:
these DQM changes got in the way - could you please remove them, leaving only the uptate to the mvaId plot range?

Reply:
please avoid fixing the conflict in nanoAOD_cfi manually, it's better to rebase on top of the current master branch because the DQM file got fixed in the meanwhile